cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -93,19 +93,19 @@ class CrossEntropyResult:
     May also include results from speckle purity benchmarking (SPB) performed
     concomitantly.
 
-    Attributes:
+    Args:
```

Comment:
This section describes the attributes of the dataclass, so "Args" looks confusing. Is there a tool that complains about this?

Reply:
It looks like the problems were with the multiple indenting, not the section title, so I changed it back to "Atteributes:".  
Thanks for the quick review.

Reply:
We could experiment with the [docparams and docstyle](https://github.com/PyCQA/pylint/blob/master/pylint/extensions/docparams.rst) pylint plugins.